fixtures/minuit-une/ivl-photon.json

```diff
@@ -0,0 +1,798 @@
+        },
+        {
+          "dmxRange": [13, 122],
+          "type": "Rotation",
+          "comment": "Rotation fast CCW"
+        },
+        {
+          "dmxRange": [123, 133],
+          "type": "Rotation",
+          "comment": "Rotation stop"
+        },
+        {
+          "dmxRange": [134, 242],
+          "type": "Rotation",
+          "comment": "Rotation fast CW"
+        },
+        {
+          "dmxRange": [243, 255],
+          "type": "Rotation",
+          "comment": "Index"
+        }
+      ]
+    },
+    "Gobo Beam Size": {
+      "fineChannelAliases": ["Gobo Beam Size fine"],
+      "defaultValue": 65535,
+      "highlightValue": 65535,
+      "capability": {
+        "type": "WheelSlot",
+        "comment": "Beam 360deg"
+      }
+    },
+    "Reflector Zoom": {
+      "fineChannelAliases": ["Reflector Zoom fine"],
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Zoom",
+          "comment": "Mirror"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "Zoom",
+          "comment": "Reflector"
+        }
+      ]
+    },
+    "Reflector Speed": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Speed",
+          "comment": "Slow"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "Speed",
+          "comment": "Fast"
+        }
+      ]
+    },
+    "Tilt 1": {
+      "fineChannelAliases": ["Tilt 1 fine"],
+      "defaultValue": 32767,
+      "highlightValue": 32767,
+      "capabilities": [
+        {
+          "dmxRange": [0, 32767],
+          "type": "Tilt",
+          "angleStart": "0%",
+          "angleEnd": "100%",
+          "comment": "Frost",
+          "menuClick": "center"
+        },
+        {
+          "dmxRange": [0, 65535],
+          "type": "Tilt",
+          "angleStart": "0%",
+          "angleEnd": "100%",
+          "comment": "Tilt Centre",
+          "menuClick": "center"
+        },
+        {
+          "dmxRange": [32767, 65535],
+          "type": "Tilt",
+          "angleStart": "0%",
+          "angleEnd": "100%",
+          "comment": "Reflective",
+          "menuClick": "center"
+        }
+      ]
+    },
+    "Dimmer 1": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "Intensity",
+        "comment": "Intensity off…bright"
+      }
+    },
+    "Red 1": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "comment": "Red off…bright"
+      }
+    },
+    "Green 1": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "comment": "Green off…bright"
+      }
+    },
+    "Blue 1": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "comment": "Blue off…bright"
+      }
+    },
+    "Tilt 2": {
```

Comment:
Here we have 8 identical sets of 6 channels, and I think OFL's "template channels" can make this a lot more concise, but I haven't used them myself so I'm not sure of the syntax for this case.